ASWF/meetings/tsc/2020-11-09.md

```diff
@@ -0,0 +1,251 @@
+          the question as: is this a good idea for color management? Most on 
+          same page to not recommend it. If they are going to do it, and ACES 
+          config has it, and should be called what they are.
+        - Mark B: Color spaces are a simplification to allow talking about 
+          OCIO. Just a block in chain that joins images together.
+        - Carol: Should be explicit though, call this what it is, not a color 
+          space.
+        - MArk B: Increasing complexity with little benefit.
+        - Matthias: Sort of agree with Carol. Agree having them in there as 
+          first class citizen potentially perpetuating workflow and simplistic 
+          interpretation of color management to avoid. Don't like to 
+          compromise, but if have to have it, see benefit of it being explicit.
+        - Mark B: Who needs it other than the config author? If you're the 
+          author you know where you need to_ref and from_ref. Current toolkit 
+          not complex and solves all these problems without extra level.
+        - Carol: Another group pf people sit between authors and users. Those 
+          that don't have someone to write config for them and need to 
+          understand the config. This would aid those people and it's a big. 
+          Not everyone can be a config author, requires skill not all have. 
+          Need to support them.
+        - J: The question is what is OCIO, who does it serve? A lot of that 
+          user experience is driven by applications that use OCIO. What do we 
+          want to enable those workflows to be like in OCIO. If there's a need 
+          for these, that's a thing. But does OCIO have an opinion about this? 
+          Or do we sacrifice some of that to give users accessible use of OCIO?
+        - Matthias: Look at it from different point of view. Having transform 
+          is one thing. Doug, any idea what would look like when presented to 
+          end user, in say... Nuke. OCIOColorSpace node would not have this. 
+          Doesn't fit there. Would it be a separate thing? Just an OCIO 
+          transform or legacy transform?
+        - Doug: Will vary based on application and task. For example, if 
+          importing something into proper color managed env. If using Maya and 
+          ACEScg as rendering space. May want to bring something in and call 
+          it ACEScg, but you're not sure what the original texture is. Don't 
+          have info so going to apply inverse gamma curve and call it ACEScg. 
+          Same workflow as textures where you do know color space. May have 
+          other scenarios where want to apply transform itself like 
+          FileTransform.
+        - Kevin: If I know how to read config, and just want to get transform, 
+          this is a convenience. File transforms don't encapsulate all new 
+          transforms, so harder to put this in file.
```

Comment:
Which new transforms aren't encapsulated by CTF?

Reply:
Zach, you are correct, every transform in OCIO may be serialized as CTF.